src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -33,17 +33,15 @@ public class ModuleIntrinsics
         [TraceSource("Modules", "Module loading and analysis")]
```

Comment:
@iSazonov, your last commit had 10 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help -SourcePath for all PowerShell modules..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```